fairlearn/reductions/_exponentiated_gradient/exponentiated_gradient.py

```diff
@@ -35,37 +35,43 @@ class ExponentiatedGradient(BaseEstimator, MetaEstimatorMixin):
         violation is at most :code:`2*(eps+best_gap)`
     :type eps: float
 
-    :param T: Maximum number of iterations
-    :type T: int
+    :param max_iter: Maximum number of iterations
+    :type max_iter: int
 
     :param nu: Convergence threshold for the duality gap, corresponding to a
         conservative automatic setting based on the statistical uncertainty in measuring
         classification error
     :type nu: float
 
-    :param eta_mul: Initial setting of the learning rate
-    :type eta_mul: float
+    :param learning_rate: Initial setting of the learning rate
+    :type learning_rate: float
```

Comment:
i'm not sure how i feel about renaming this one... I know that this better matches the current description, but the actual initial learning rate is `eta_mul / B`. Plus, `sklearn` is also using `eta0` in this context: https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.SGDClassifier.html

Reply:
okay, if this is commonly used we can do `eta0`. I just didn't like it to be `mul` because that kind of conflicts with the "initial setting" description. I expected it to be multiplied with `eta` (learning rate). `eta0` makes intuitively sense.

Reply:
this is fine...